### ISSUE TITLE
Force relative URLs for configured domain when users insert links (configurable)

### ISSUE THREAD

**Issue by jnnkB** [GetmeUK/contenttools-2-roadmap#6]
If a link is added that refers to the page it self, it should automatically remove the domain and become a relative link so it's still usable if the domain changes or a site has multiple domains with the content.

This could be implemented on the server side but I think it should be implemented on the client side.

I think it would make sense to clear up what belongs to the client and what belongs to the server.

**Comment by anthonyjb** [comment]
Agreed this would be a really nice feature to add and something we've had issues with before too actually. We'd need to potentially allow it to be a setting for the editor but yes we could force all links to be relative client-side for a configured domain (which can default to the current domain, for wildcard domains I think this should be configured.)
